src/app.js

```diff
@@ -38,8 +37,8 @@ export default function (app) {
     }
 
     if (obj = plugin.hooks) {
-      Object.keys(obj).forEach(function (key) {
-        hooks[key].push(obj[key])
+      Object.keys(obj).map(function (key) {
```

Comment:
@leeoniya I am awful at taking notes, so if I don't push this code, I'll forget about it later. But you're right, obsessions don't lead anywhere good. 👍